environment-pipelines/buildspec-install-build-tools.yml

```diff
@@ -13,6 +13,12 @@ phases:
     commands:
       - set -e
       - echo "Deploying ${APPLICATION} environments"
+      - |
+          if [ ! -f .copilot-version ]; then
+            echo "Cannot find .copilot-version file"
+            exit 1
+          fi
+      - COPILOT_VERSION=`cat .copilot-version`
```

Comment:
Do any projects not have a `.copilot-version` file?

Could this be a breaking change?

Reply:
This file predates this pipeline, it will already be present in deploy repositories